app/views/support.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Cookies - NHS prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {{ breadcrumb({
+    href: "/",
+    text: "Home"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1 id="support">Support</h1>
+      <p>The NHS prototype kit is maintained by <a href="https://service-manual.nhs.uk/service-manual-team">NHS England service manual team</a> with help from community contributors. If you&#39;ve got a question or need support you can:</p>
+      <ul>
+          <li><a href="mailto:england.service-manual@nhs.net?subject=NHS%20prototype%20kit%20-%20About">email the service manual team</a></li>
+          <li>use Slack - if you have an NHS England email or have been given access to the NHS England slack instance you can use the <a href="https://nhsdigitalcorporate.enterprise.slack.com/archives/C042J3MTJG2">NHS England #prototype-kit slack channel</a>. If you do not you can also use the <a href="https://nhs-service-manual.slack.com/messages/CFYL2GDGW"> NHS digital service manual #prototype-kit Slack channel</a></li>
+          <li>add a bug or view issues on on GitHub for the <a href="https://github.com/nhsuk/nhsuk-prototype-kit/issues">prototype kit</a> or  <a href="https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/issues">this documentation site</a>  </li>
+        </ul>
```

Comment:
This bullet list of 3 items containing 5 links feels a bit overwhelming to me. 

I wonder if we could use a pattern more like [Contact a department or team](https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/) with headings?

Also think for GitHub we can just direct everyone to open issues on the main prototype kit repo here: https://github.com/nhsuk/nhsuk-prototype-kit/issues/new/choose - and if they’re to do with the documentation website then we can just move them across? (Saves people having to think about which is which)

Something like this?

```suggestion
      <h2>Email</h2>
      <p><a href="mailto:england.service-manual@nhs.net?subject=NHS%20prototype%20kit%20-%20About">england.service-manual@nhs.net</a></p>
      
      <h2>Slack</h2>
      
      <p><a href="https://nhsdigitalcorporate.enterprise.slack.com/archives/C042J3MTJG2">#prototype-kit channel</a> on the NHS England Slack.</p>
      
      <p><a href="https://nhs-service-manual.slack.com/messages/CFYL2GDGW">#prototype-kit channel</a> on the NHS digital service manual Slack</p>
      
      <h2>GitHub</h2>
      
      <p><a href="https://github.com/nhsuk/nhsuk-prototype-kit/issues/new/choose">Report a bug or issue</a> - requires a free GitHub account.</p>
```